checkoutmanager/runner.py

```diff
@@ -148,11 +163,7 @@ def main():
         return
 
     custom_actions = get_custom_actions()
-
-    executor = get_executor(options.single)
-    for dirinfo in conf.directories(group=group):
-        executor.execute(execute_action, (dirinfo, custom_actions, action))
-    executor.wait_for_results()
+    executor = run(action, group, custom_actions, conf, options.single, False)
```

Comment:
See the comment on `run()` for reducing the number of options.

Additionally, the list is quite long. To make it clearer, please use the keyword arguments (`custom_actions=custom_actions`) instead of only passing them in the right order. This makes it easier if we want to change something later on.